src/gui/dialogs/network_transmission.cpp

```diff
@@ -113,6 +113,9 @@ void network_transmission::pre_show(window& window)
 		subtitle_label.set_use_markup(true);
 	}
 
+	// NOTE: needed to avoid explicit calls to invalidate_layout()
+	// in network_transmission::pump_monitor::process()
+	find_widget<label>(&window, "numeric_progress", false).set_label(" ");
```

Comment:
What is this doing?

Reply:
I honestly have no idea. The WML set the label to a single space with this comment, but that makes it fail validation, so I figured setting it in the C++ would have the same effect.

Reply:
Given there's no longer any explicit call to invalidate_layout in network_transmission::pump_monitor::process(), is it even still needed?

Reply:
That's a good question! The comment suggests removing this would mean we need to add explicit calls there, but I have no idea why a single space would make that difference.

Reply:
Looks like https://github.com/wesnoth/wesnoth/commit/34d45d3ee5ff4d3ec939e5ae0272a292eefce60f added this in WML.